dockerfile_functions.sh

```diff
@@ -411,15 +416,22 @@ generate_dockerfile() {
 	pkg=$2
 	bld=$3
 	btype=$4
-	os=$5
+	case $5 in
+		windows*)
+			os=windows
+			distro=$5 ;;
```

Comment:
Can you please change `os` to `os_family` and `distro` to `os`. This is to keep in consistent across all shell scripts. Same for other instances. Thanks